unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -69,6 +74,54 @@ class TimeTest {
     assertTrue(now_time_t > 1502752688)
   }
 
+  @Test def strftimeDoesNotReadMemoryOutsideStructTm(): Unit = {
+    Zone { implicit z =>
+      if (sizeof[tm] < 56.toULong) {
```

Comment:
@WojciechMazur Thank you for the timely & helpful review.

OK, I will use the Big Hammer in two places, strftime() & strptime(), rather than trying to 
guess what future changes may bring.